scene/main/viewport.cpp

```diff
@@ -881,7 +881,7 @@ void Viewport::_process_picking() {
 					}
 				}
 			}
-		} else {
+		} else if (!gui.mouse_over) {
```

Comment:
In #61088 the following behavior got introduced:
- When an `InputEventMouseMotion` is over a Control node and the Control node doesn't mark the event as handled, then the input event can be used for physics picking. See

https://github.com/godotengine/godot/blob/7827c8e37010241b376748b22db85847c8614dbd/scene/main/viewport.cpp#L2001-L2004

With the changes of this PR in its current form, physics picking will no longer work for these input events in 3D, because for these input events, `gui.mouse_over == true`.

Reply:
Thanks for the explanation, I understand and I will take another look.  Also, I will not include the .godot folder going forward.